src/lib.rs

```diff
@@ -303,26 +303,23 @@ pub fn panic_hook(panic_info: &PanicInfo<'_>) {
 }
 
 pub fn handle_force_redraws(app: &mut App) {
-    // Currently we use an Option... because we might want to future-proof this
-    // if we eventually get widget-specific redrawing!
-    if app.proc_state.force_update_all {
-        update_all_process_lists(app);
-        app.proc_state.force_update_all = false;
-    } else if let Some(widget_id) = app.proc_state.force_update {
-        update_final_process_list(app, widget_id);
-        app.proc_state.force_update = None;
+    for proc in app.proc_state.widget_states.values_mut() {
+        if proc.force_update {
+            // NB: Currently, the "force update" gets fixed in the draw.
+            proc.update_displayed_process_data(&app.data_collection);
+        }
     }
```

Comment:
Basically only update if force_update gets called. Currently a bit disjointed (see NB), ideally we join the two together after such that a force update updates the data rendered/stored *and* renders it, then unsets the force update flag.